File: java/src/test/java/org/hello/HelloMessageTest.java
package org.hello;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class HelloMessageTest {

  @Test
  void shouldReturnHelloWorld() {
    var message = new HelloMessage();
    assertThat(message.getText()).isEqualTo("Hello World!");
  }
}

File: kotlin/src/test/kotlin/org/hello/HelloMessageTest.kt
package org.hello

import io.kotest.core.spec.style.DescribeSpec
import io.kotest.matchers.shouldBe

class HelloMessageTest : DescribeSpec({

  describe("HelloMessage") {
    it("should return hello world") {
      val message = HelloMessage()
      message.text shouldBe "Hello World!"
    }
  }
})
